packages/swipe/index.vue

```diff
@@ -115,9 +134,10 @@ export default create({
       // reset offset when children changes
       clearTimeout(this.timer);
       this.width = this.$el.getBoundingClientRect().width;
+      this.height = this.$el.getBoundingClientRect().height;
```

Comment:
const rect = this.$el.getBoundingClientRect();
this.width = rect.width;
this.height = rect.height;